### PR TITLE
[LM-9405] Added post detail screen

### PR DESCRIPTION
**Pull request by prabha-likeminds**
## Description

**Features**
1.  Added post detail screen with functionality

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [x] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [x] Label added
- [x] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [x] Description about the Feature/ Bugs added

### CODE REVIEW

File: FeedSX/store/actions/createPost.ts [hunk 1]
```diff
@@ -47,13 +47,14 @@ export const addPost = (payload?: any) => async (dispatch: Dispatch) => {
...
-      type: UPLOAD_ATTACHMENTS,
-      body: payload
-    });
-  } catch (error) {
-    Alert.alert(`${error}`);
-  }
-};
\ No newline at end of file
+export const setUploadAttachments =
+  (payload?: any) => async (dispatch: Dispatch) => {
```
Comment (ishaan1607) [inline FeedSX/store/actions/createPost.ts:51]: why `any`?

File: FeedSX/store/actions/toast.ts [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
-    return await dispatch({
-      type: SHOW_TOAST,
-      body: payload
-    });
-  } catch (error) {
-    Alert.alert(`${error}`);
-  }
-};
+export const showToastMessage =
+  (payload?: any) => async (dispatch: Dispatch) => {
```
Comment (ishaan1607) [inline FeedSX/store/actions/toast.ts:7]: why `any`?

**Review by ishaan1607 — CHANGES_REQUESTED** [review]

File: FeedSX/store/actions/createPost.ts (hunk 1 above)
Comment (prabha-likeminds) [reply FeedSX/store/actions/createPost.ts:51]: this is related to dispatch as any, have to define according to that.

File: FeedSX/store/actions/toast.ts (hunk 2 above)
Comment (prabha-likeminds) [reply FeedSX/store/actions/toast.ts:7]: this is related to dispatch as any, have to define according to that.

**Review by ishaan1607 — APPROVED** [review]

**Pull request merged**